relayer/application_relayer.go

```diff
@@ -209,7 +229,7 @@ func (r *ApplicationRelayer) relayMessage(
 	startCreateSignedMessageTime := time.Now()
 	// Query nodes on the origin chain for signatures, and construct the signed warp message.
 	var signedMessage *avalancheWarp.Message
-	if useAppRequestNetwork {
+	if r.sourceBlockchain.UseAppRequestNetwork() {
```

Comment:
Nit: Should we check `if r.warpClient != nil` here instead? It looks correct as is, but the two independent checks on `r.sourceBlockchain.UseAppRequestNetwork()` feel error prone for prevent a nil pointer dereference.